cryptorand/cipher_test.go

```diff
@@ -0,0 +1,30 @@
+package cryptorand_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/coder/coder/cryptorand"
+)
+
+func TestCipherAES256(t *testing.T) {
```

Comment:
Should add a test to ensure a unique nonce is being generated each time we encrypt something new, and another to ensure that changing the nonce causes decryption to fail

Reply:
Should add another test to ensure that the format doesn't change in a backwards incompatible way. Hardcode a key, encrypt something once and then hardcode the encrypted result to ensure it gets decrypted successfully and matches the same input string.